gravitee-am-test/api/config/ci.config.js

```diff
@@ -13,7 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export interface BasicAuthentication {
-  username: string;
-  password: string;
-}
+module.exports = {
```

Comment:
I've reduced the config files to two in order to keep things simple. There needs to be two files per config for Jest: this one is passed as the `--config=` CLI argument and it points to another file `ci.setup.js` which contains the actual test environment variables.